integration-tests/src/router.rs

```diff
@@ -6,7 +6,11 @@ use crate::polkadot_test_net::*;
 use std::convert::Into;
```

Comment:
How about adding an integration test to include all pool types?

Reply:
Do you mean to test a trade with all pool types?